tests/model/test_screening.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import absolute_import, division, print_function
+import ispyb.model.datacollection
+import ispyb.model.screening
+import ispyb.model.__future__
+
+
+def test_model_screening(testdb, testconfig):
+    ispyb.model.__future__.enable(testconfig)
+    dc = ispyb.model.datacollection.DataCollection(1052494, testdb)
+    assert len(dc.screenings) == 7
+
+    for screening in (
+        dc.screenings[2],
+        ispyb.model.screening.Screening(dc.screenings[2]._screening_id, testdb),
+    ):
+        assert screening.program_version == "EDNA MXv1"
```

Comment:
would `.program` be clearer than `.program_version`, given that the example string includes the program name?